src/analyze/meson.build

```diff
@@ -30,11 +30,10 @@ systemd_analyze_sources = files(
 )
 
 tests += [
-        [files('test-verify.c',
-               'analyze-verify-util.c',
-               'analyze-verify-util.h'),
-         [libcore,
-          libshared],
-         [],
-         core_includes],
+        { 'sources' : files(
+                'test-verify.c',
+                'analyze-verify-util.c',
+                'analyze-verify-util.h'),
```

Comment:
What about 4a7ee0a5217ab963c12704dbff428c43d9df81f0? You dropped .h files from the file lists… I think we should do the same here.

Reply:
Well, regexes don't catch everything :D